test/cli_network_test.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"runtime"
+	"strings"
+
+	"github.com/alibaba/pouch/test/command"
+	"github.com/alibaba/pouch/test/environment"
+	"github.com/go-check/check"
+	"github.com/gotestyourself/gotestyourself/icmd"
+)
+
+// PouchNetworkSuite is the test suite fo network CLI.
+type PouchNetworkSuite struct{}
+
+func init() {
+	check.Suite(&PouchNetworkSuite{})
+}
+
+// SetUpSuite does common setup in the beginning of each test suite.
+func (suite *PouchNetworkSuite) SetUpSuite(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+	command.PouchRun("pull", busyboxImage).Assert(c, icmd.Success)
+}
+
+// TestNetworkDefault tests default bridge network is created.
```

Comment:
default bridge network is created. -> the creation of default bridge network.

Reply:
ok~